### ISSUE TITLE
is the network connection issue still ongoing?

### ISSUE THREAD

**Issue by wawaawawa** [sotch-pr35mac/syng#237]
i found the last time someone mentioned it in 2020, but i can't seem to get audio to play from the app

**Comment by wawaawawa** [comment]
mac os version by the way

**Comment by sotch-pr35mac** [comment]
Yes, it looks like the service used for audio files has gone offline, and the feature has been removed for now until a suitable alternative can be found.
